ignite/distributed/utils.py

```diff
@@ -1,9 +1,11 @@
 import socket
+from contextlib import contextmanager
 from functools import wraps
 from typing import Any, Callable, List, Mapping, Optional, Tuple, Union
 
 import torch
 
+import ignite.distributed as idist
```

Comment:
In utils.py you do not need to reimport idist as all functions are defined in this module. see below `__all__`. By the way, we have to add new function in `__all__` as well

Reply:
yes, I see now thats all funcs are already here, no need to import idist
should we sort __all__ alfabetikaly or not?
```
__all__ = [
    "backend",
    "broadcast",
    "device",
    "available_backends",
    "model_name",
    "get_world_size",
    "get_rank",
    "get_local_rank",
    "get_nproc_per_node",
    "get_node_rank",
    "get_nnodes",
    "spawn",
    "initialize",
    "finalize",
    "show_config",
    "set_local_rank",
    "all_reduce",
    "all_gather",
    "barrier",
    "hostname",
    "has_xla_support",
    "has_native_dist_support",
    "has_hvd_support",
    "sync",
    "registered_computation_models",
    "one_rank_only",
    "new_group",
    "one_process_first"
]```

Reply:
> should we sort all alfabetikaly or not?

Let's keep it as it is. We can sort it alphabetically in another PR if needed